source/includes/_groups.md

```diff
@@ -0,0 +1,366 @@
+# Groups
+
+A group is an entity comprising several users. When users are added to a group, they benefit from the permissions, quotas and limits of this group.
+
+Through the management API, it is possible, for the current domain, to:
+
+- list groups,
+- lookup a specific group,
+- create groups,
+- grant groups permissions, quotas and limits,
+- list, add and delete users in groups,
+- delete groups
+
+## The Group object
+
+> Example object
+
+```json
+{
+    "title": "My Users",
+    "group_id": "my-users",
+    "user_count": 1,
+    "permissions": [
+        "publish_dataset",
+        "edit_dataset"
+    ],
+    "limits": {
+        "max_records_by_dataset": 1000000,
+        "max_datasets": 500
+    },
+    "quotas": {
+        "limit": 1000,
+        "unit": "day"
+    }
+}
+```
+
+Attribute | Description
+--------- | -----------
+`title` <br> *string*      | The group title
+`group_id` <br> *string*   | The group identifier
+`user_count` <br> *number* | The count of users in the group
+`permissions` <br> *array* | A list of permissions granted to these group members
+`limits` <br> *object*     | An object holding the group's limits on this domain
+`quotas` <br> *object*     | An object holding the group's quotas on this domain
```

Comment:
Should we document them as "the limits object" and "the quotas object" to make it clearer? I really don't know if it would be better or not

Reply:
Definitely. We still need to document these two but as soon as they are documented, we will do that.